### ISSUE TITLE
Stylesheet: Property view rows are always visible on macOS with classic theme

### ISSUE THREAD

**Issue by maxwxyz** [FreeCAD/FreeCAD#15799]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

On macOS the different colored rows in the property view are always visible, even when nothing is selected or no object is in the document. This does not happen on Windows.

<img width="1677" alt="Bildschirmfoto 2024-08-06 um 20 11 23" src="https://github.com/user-attachments/assets/61e7dd79-30e2-49d7-8833-205acf6fdf76">

### Full version info

```shell
OS: macOS 13.6.7
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38419 (Git)
Build type: Release
Branch: main
Hash: 270bb0122ea15bbc562942b9d0756fabb89362c0
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * freecad.gears 1.2.0
  * OpenTheme 2024.7.30
  * CfdOF 1.27.4
  * Defeaturing 1.2.2
  * Manipulator 1.5.7
  * fasteners 0.5.25
  * CurvedShapes 1.0.9
  * sheetmetal 0.4.23
  * Curves 0.6.42
```

### Subproject(s) affected?

Core

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
When selecting an empty body, the rows are rendered differently where there is actually a property:
<img width="382" alt="Bildschirmfoto 2024-08-06 um 20 13 40" src="https://github.com/user-attachments/assets/09562d7c-5363-427c-b387-b45d22d7d895">

**Comment by maxwxyz** [comment]
@MisterMakerNL any news on this?

**Comment by MisterMakerNL** [comment]
I cannot test atm.
Coul;d you check opentheme if it is also there?

**Comment by luzpaz** [comment]
Hey @johnocfii can you test on macos ?

**Comment by maxwxyz** [comment]
@MisterMakerNL same issue with OpenTheme

@kadet1090 @obelisk79 FYI

**Comment by obelisk79** [comment]
> @MisterMakerNL same issue with OpenTheme
> 
> @obelisk79 FYI


I am unable to recreate this on OpenTheme

**Comment by MisterMakerNL** [comment]
@JohnOCFII Do you have most current opentheme, since these rows are recently added by @kadet1090 as previous they couldn't be style which resulted in inverse problem of never having a background.

**Comment by JohnOCFII** [comment]
> @JohnOCFII Do you have most current opentheme, since these rows are recently added by @kadet1090 as previous they couldn't be styled which resulted in inverse problem of never having a background.

Yes, I've got the most recent version of OpenTheme.  What I don't know is if some combination of overlay or transparent window control could influence this behavior.

Odd that this Add-on description window doesn't show the version number, but the main Add-on list of items does.
<img width="980" alt="Screenshot 2024-08-24 at 10 39 32 AM" src="https://github.com/user-attachments/assets/7508024d-7b0c-46ff-90cd-22673f173ae4">

**Comment by kadet1090** [comment]
This is strictly related to the MacOS QtStyle probably, no way to do anything about it without changing the style.

**Comment by maxwxyz** [comment]
@kadet1090 @MisterMakerNL would https://github.com/FreeCAD/FreeCAD/pull/16042 fix this too?

**Comment by maxwxyz** [comment]
@MisterMakerNL @kadet1090 ^

**Comment by kadet1090** [comment]
@maxwxyz you are the one with MacOS - try it. It probably will for non-classic preincluded themes.

**Comment by maxwxyz** [comment]
@kadet1090 still an issue on the latest weekly
```
OS: macOS 13.6.9
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38622 (Git)
Build type: Release
Branch: main
Hash: ed97f3c4591303ee3b54906768f8f4eba5211265
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * freecad.gears 1.2.0
  * OpenTheme 2024.8.17
  * CfdOF 1.27.5
  * Defeaturing 1.2.2
  * Manipulator 1.5.7
  * fasteners 0.5.27
  * CurvedShapes 1.0.9
  * sheetmetal 0.4.24
  * Curves 0.6.45
```

**Comment by luzpaz** [comment]
@hyarion can you reproduce ?

**Comment by hyarion** [comment]
Yes I can reproduce. This is one of the reasons for https://github.com/FreeCAD/FreeCAD/pull/17410#issuecomment-2431381751

**Comment by adrianinsaval** [comment]
I don't know if we can truly consider this fixed, we rather just hid the option that led to this in the startpage

**Comment by MisterMakerNL** [comment]
This could probably also be fixed by switching classic Qtstyle too fusion. But the point of classic is that it adapts too the OS Qtstyle. And so it kinda is absolete if we don't do that anymore.
Also fixing bugs in a stylesheet is waaaaay easier than doing it in classic.
So I'm very satisfied on how it is now.

**Comment by maxwxyz** [comment]
Can someone test on mac + Qt6?

**Comment by maxwxyz** [comment]
Removing the blocker tag as for 1.0 it was "fixed" with the workaround

**Comment by maxwxyz** [comment]
deprecated theme
